<launch>

<!-- - - - - - - - - - - - - - - - - - - - UAV 1 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
    <node pkg="offboard_control" name="Quad1_action" type="shield_action.py" output="screen">
        <param name="quad_ros_namespace" value="/uav1" />
        <param name="xOffset" value="4.5" />
        <param name="yOffset" value="2.5" />
        <param name="zOffset" value="0" />
        <param name="freq" value="20" />
        <param name="time_full_traj" value="10" />
    </node>

<!-- - - - - - - - - - - - - - - - - - - - UAV 2 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
    <node pkg="offboard_control" name="Quad2_action" type="shield_action.py" output="screen">
        <param name="quad_ros_namespace" value="/uav2" />
        <param name="xOffset" value="6" />
        <param name="yOffset" value="0" />
        <param name="zOffset" value="0" />
        <param name="freq" value="20" />
        <param name="time_full_traj" value="10" />
    </node>

<!-- - - - - - - - - - - - - - - - - - - - UAV 3 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
    <node pkg="offboard_control" name="Quad3_action" type="shield_action.py" output="screen">
        <param name="quad_ros_namespace" value="/uav3" />
        <param name="xOffset" value="8.7" />
        <param name="yOffset" value="2.5" />
        <param name="zOffset" value="0" />
        <param name="freq" value="20" />
        <param name="time_full_traj" value="10" />
    </node>


</launch>
